src/main/resources/templates/owners/findOwners.html

```diff
@@ -26,6 +26,28 @@ <h2>Find Owners</h2>
           Owner by Last Name</button>
       </div>
     </div>
+
+      <form th:object="${owner}" th:action="@{/owners}" method="get"
```

Comment:
Add check here like below form for example 

so add `th:block th:if="${toggles[4] == true}"`